sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -137,27 +130,11 @@ export interface CertificateOperationError {
     readonly message?: string;
 }
 
+// Warning: (ae-forgotten-export) The symbol "CertificatePolicyProperties" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "PolicySubjectProperties" needs to be exported by the entry point index.d.ts
```

Comment:
`CertificatePolicyProperties` and `PolicySubjectProperties` need to be exported.

Reply:
Done.